PHP String Functions money_format(), Closes #3218

reganmarie opened the pull request:
<!--
👋 Hi, thanks for sending a PR to Codecademy Docs! 💖
Please fill out all fields below to ensure your PR is reviewed quickly.

**IMPORTANT**

If you would like to receive credit for your contribution to an entry, make sure to have your Codecademy user profile linked to your GitHub account:

1. Go to your Codecademy dashboard.
2. Click on your profile image in the top-right and then choose "Profile".
3. Then click "Edit Profile".
4. In the "GitHub Username" field, add your username (without the @).
5. Click "Save Changes"

Of course, you can opt not to do this and be listed as an "Anonymous contributor", instead. :)
-->

### Description

- Closes #3218 
- Created money_format.md on filepath docs/content/php/concepts/string-functions/terms/money-format/money-format.md
- Added Title, Description, Subjects, Tags, and Catalog Content
Added description, syntax, example, and codebyte example
- Ran through 3 online plagiarism checkers to ensure 100% unique wording and description, passed with 100%
- Followed content and style guidelines
-Checked for spelling and grammar errors
- Read over initial draft and correct for clarity and readability

### Type of Change

<!--- Please delete or cross off the bullet point(s) that are irrelevant to this PR: -->

- Adding a new entry

### Checklist

<!-- Please check ALL the boxes: -->

- [X] All writings are my own.
- [X] My entry follows the Codecademy Docs style guide.
- [X] My changes generate no new warnings.
- [X] I have performed a self-review of my own writing and code.
- [X] I have checked my entry and corrected any misspellings.
- [X] I have made corresponding changes to the documentation if needed.
- [X] I have confirmed my changes are not being pushed from my forked `main` branch.
- [X] I have confirmed that I'm pushing from a new branch named after the changes I'm making.
- [X] Under "Development" on the right, I have linked any issues that are relevant to this PR (write "Closes #<issue number> in the "Description" above).

<!--
Having trouble with the PR checker? Here are some common issues and resolutions:

- verify_formatting is failing
  - run `yarn format path/to/markdown/file.md` or `yarn format:all` and commit the results
- verify_lint is failing
  - same as above
  - if verify_lint is still failing, running `yarn lint` locally should let you know what needs to be changed by hand
- test is failing
  - ensure any new markdown files have a `Title` and `Description` defined in their metadata
  - ensure any new markdown files only contain alphanumerics and dashes in their file names and have the same name as their parent directory
  - if that looks ok, running `yarn test` locally should let you know what the issue is
-->

SSwiniarski reviewed (approved):
@reganmarie I added the "Obsolete" tag and this is ready for a second review.

caupolicandiaz commented on content/php/concepts/string-functions/terms/money-format/money-format.md:
```diff
@@ -0,0 +1,92 @@
+
+- `=f`: The character to use for padding. If left empty, the default padding is a space.
+- `^`: Disables grouping characters. Defined by the current locale, if it is used.
+- `+` or `(`: How to format positive and negative numbers. If `(` is specified, the negative numbers will be enclosed by parenthesis. If `+` is specified, there will be no sign in front of the positive numbers and a `-` in front of the negative numbers. If current locale is used in conjunction with `+`, the current locale's version of `+` and `-` will be used. The default is `+`.
```
```suggestion
- `+` or `(`: How to format positive and negative numbers. If `(` is specified, the negative numbers will be enclosed by parenthesis. If `+` is specified the current locale's version of `+` and `-` will be implemented (this is the default). 
```

caupolicandiaz commented on content/php/concepts/string-functions/terms/money-format/money-format.md:
````diff
@@ -0,0 +1,92 @@
+```php
+<?php
+  $value = 7269.30
+  set_locale(LC_MONETARY, 'en_US');
````
```suggestion
  setlocale(LC_MONETARY, 'en_US');
```

caupolicandiaz commented on content/php/concepts/string-functions/terms/money-format/money-format.md:
````diff
@@ -0,0 +1,92 @@
+US International format is: USD 7,269.30
+```
+
+## Codebyte Example
````
The docs compiler does currently support this function, but it is likely that it will not for much longer. I suggest removing the codebyte since this is a deprecated function

caupolicandiaz commented on content/php/concepts/string-functions/terms/money-format/money-format.md:
```diff
@@ -0,0 +1,92 @@
+  - 'paths/computer-science'
+---
+
+The **`money_format()`** function takes a given number and formats it to a currency string. `money_format()` is often used in conjunction with `set_locale()`, and if additional formatting specifications are not given, `money_format()` will default to the currency format of the location specified in `set_locale()`.
```
```suggestion
The **`money_format()`** function takes a given number and formats it to a currency string. `money_format()` is often used in conjunction with `setlocale()`, and if additional formatting specifications are not given, `money_format()` will default to the currency format of the location specified in `setlocale()`.
```

caupolicandiaz commented on content/php/concepts/string-functions/terms/money-format/money-format.md:
```diff
@@ -0,0 +1,92 @@
+
+## Example
+
+This example uses the `money_format()` function in conjunction with `set_locale()` to format a number to the international format for the United States.
```
```suggestion
This example uses the `money_format()` function in conjunction with `setlocale()` to format a number to the international format for the United States.
```

caupolicandiaz reviewed (changes requested):
@reganmarie I've added my comments. Please update the entry, and tag me when you've finished making changes. Thanks

reganmarie commented:
@caupolicandiaz Made all of the changes requested!

Updated + and ( flag description in line 37
Changed set_locale() to setlocale() in lines 70, 16, and 65.
Removed codebyte example since the compiler will likely no longer support this function soon.
Ran lint to ensure proper formatting

caupolicandiaz reviewed (approved):
@reganmarie @SSwiniarski Looks good to go

reganmarie commented:
Thank you @caupolicandiaz and @SSwiniarski ! 😊
